adapters/Unity/UnityAdapter/AdNetworkAdapterProxy/GADMUnityBannerNetworkAdapterProxy.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC.
```

Comment:
For all the files that were created before this PR, the year should remain the same.
This applies to all the other files.